src/Control/Monad/Aff/Bus.purs

```diff
@@ -33,6 +34,7 @@ import Prelude
 
 import Control.Monad.Aff (Aff, attempt, launchAff_)
 import Control.Monad.Aff.AVar (AVAR, AVar, killVar, makeEmptyVar, putVar, takeVar)
+import Control.Monad.Eff.AVar (isKilledVar)
```

Comment:
Can you merge the import with the one above here?

Reply:
sure

Reply:
Oh, nevermind, sorry - one is Eff one is Aff :grin:

Reply:
:d